src/test/java/org/broadinstitute/hellbender/tools/walkers/GenotypeGVCFsUnitTest.java

```diff
@@ -121,9 +122,17 @@ public void testPhasingUpdate(){
         Assert.assertEquals(homVarAfterCleanup.getAnyAttribute(GATKVCFConstants.HAPLOTYPE_CALLER_PHASING_GT_KEY), GenotypeGVCFs.PHASED_HOM_VAR_STRING);
         final Genotype homRefAfterCleaning = GenotypeGVCFs.cleanupGenotypeAnnotations(withPhasing, true).get(1);
         Assert.assertEquals(homRefAfterCleaning.getAnyAttribute(GATKVCFConstants.HAPLOTYPE_CALLER_PHASING_GT_KEY), "something");
-
     }
 
-
+    @Test
+    public void testRGQ0IsNoCall(){
```

Comment:
ReferenceGenotypeQuality.  I ported this commit, which had an integration test we can't run, so I added this unit test instead.  Weirdly, none of the changes mention RGQ, so I don't know how it affects that... This unit test tests the changes that were made in the commit though.